packages/flutter/test/material/dropdown_test.dart

```diff
@@ -337,6 +339,25 @@ void main() {
     });
   }
 
+  testWidgets('Arrow icon aligns with the edge of button when expanded', (WidgetTester tester) async {
+    final Key buttonKey = new UniqueKey();
+
+    Widget build() =>
+        buildFrame(buttonKey: buttonKey, value: 'two', isExpanded: true);
+
+    await tester.pumpWidget(build());
+    final RenderBox buttonBox = tester.renderObject(find.byKey(buttonKey));
+    assert(buttonBox.attached);
+
+    final RenderBox arrowIcon =
+        tester.renderObject(find.byIcon(Icons.arrow_drop_down));
```

Comment:
definitely don't wrap this since it hides that this code is basically identical to line 349